hypothesis-python/docs/details.rst

```diff
@@ -694,3 +694,47 @@ on working with markers <pytest:mark examples>`.
 .. note::
     Pytest will load the plugin automatically if Hypothesis is installed.
     You don't need to do anything at all to use it.
+
+
+.. _fuzz_one_input:
+
+-------------------------
+Use with external fuzzers
+-------------------------
```

Comment:
I'd like this feature marked as experimental I think. I think it's a good feature that we'll probably want to stabilise at some point, but I'm not 100% about it right now and don't currently have the mental bandwidth to become so, and it seems a shame to have its current quite usable state be held back by that.